src/main/java/org/broadinstitute/hellbender/tools/spark/sv/SVUtils.java

```diff
@@ -146,6 +146,53 @@ public static int hashMapCapacity( final int nElements )
     }
 
     /**
+     * 64-bit FNV-1a hash for long's
+     */
+    public static long fnvLong64( long start, final long toHash ) {
```

Comment:
Not sure why I didn't use the 64-bit FNV-1a in the first place.  Let's use this one exclusively and kill fnvLong and fnvInt.  But now that this has moved out into independent space (rather than being encapsulated in SVKmer), I think we need a 1-arg fnvLong64 that supplies the correct initial seed:
```
public static long fnvLong64( final long toHash ) {
    return fnvLong64(14695981039346656037L, toHash);
}
```